### PR TITLE
Add Baseball Player profession

### PR DESCRIPTION
**Pull request by Nihad2004**
#### Summary

SUMMARY: Content "Add Baseball Player profession"

#### Purpose of change

Currently, there isn't any profession that starts with the baseball gear.

#### Testing

I started a new game and chose "Baseball Player" as a profession. Checked inventory and skills.
1.Start a new game. 
2.Select a scenario where you can choose all the professions.
3.Choose the "Baseball Player" profession and create the character.

#### Additional context

The profession in the profession selecting menu
image

Skills check
image

Inventory check
image

### CODE REVIEW

**Comment by Zireael07** [comment]
If I were you, I'd change the PR title to be more descriptive - e.g. "Add Baseball Player profession". This repo has A LOT of PR open at the same time and this helps the devs review them.

File: data/json/professions.json [hunk 1]
```diff
@@ -620,6 +620,25 @@
...
+  {
+    "type": "profession",
+    "id": "baseball_player",
+    "name": "Baseball Player",
+    "description": "You were playing at the local team.  With no one left to play baseball, you can eventually use your best friend for another purpose.  Home run!",
+    "points": 1,
+    "skills": [
+      { "level": 3, "name": "throw" },
+      { "level": 1, "name": "melee" }
+    ],
```
Comment (hjk321) [inline data/json/professions.json:632]:
```suggestion
    "skills": [ { "level": 3, "name": "throw" }, { "level": 1, "name": "melee" } ],
```
Arrays are usually formatted like this.

File: data/json/professions.json [hunk 2]
```diff
@@ -620,6 +620,25 @@
...
+    "items": {
+      "both": {
+        "items": [ "baseball", "tshirt", "jacket_light", "jeans", "socks", "sneakers", "helmet_ball", "sports_drink" ],
+        "entries": [ { "group": "charged_cell_phone" }, { "item": "bat", "custom-flags": [ "auto_wield" ] } ]
+      },
+      "male": [ "boxer_shorts" ],
+      "female": [ "sports_bra", "panties" ]
+    }
+  },
```
Comment (hjk321) [inline data/json/professions.json:641]:
```suggestion
  },
```
Removed extra space.

**Review by hjk321 — CHANGES_REQUESTED** [review]
Thank you for your contribution! I noticed that your JSON validation failed, so I linted it for you.

File: data/json/professions.json (hunk 1 above)
Comment (Nihad2004) [reply data/json/professions.json:632]: Thanks! Since its my frst time contributing something, I knew I am going to make lots of mistakes.

**Comment by LaVeyanFiend** [comment]
No love for ~~handegg~~ football players? 😢

P.S. Bookmark [our linter](https://dev.narc.ro/cataclysm/format.html) so the formatting issues don't happen in future PRs.

**Comment by Nihad2004** [comment]
> No love for ~handegg~ football players? 
> 
> P.S. Bookmark [our linter](https://dev.narc.ro/cataclysm/format.html) so the formatting issues don't happen in future PRs.
>
> Sure

File: data/json/professions.json (hunk 1 above)
Comment (hjk321) [reply data/json/professions.json:632]: No problem! This isn't even really a mistake, we just like to keep the files formatted a certain way.
You can go ahead and press "commit suggestion" on this thread to automatically commit it onto your branch, or "add to batch" to put both changes in a single commit.

**Comment by BrettDong** [comment]
There was a baseball player profession in the game before (see 11f3566e in #20522), and sadly it was removed in #40337. Thank you for bringing it back!

**Pull request merged**